spec.html

```diff
@@ -6211,6 +6222,8 @@
         1. Add _pending_ at the back of the Job Queue named by _queueName_.
         1. Return NormalCompletion(~empty~).
       </emu-alg>
+
+      <emu-note>For host environments consisting of multiple continents, when those host environments enqueue their own jobs, they should generally specify which continent's job queues they are enqueuing on.</emu-note>
```

Comment:
Without continents:

- HTML has event loops, each of which has an ECMAScript implementation, each of which has a set of job queues.
- When you want to run some script, you find an appropriate execution context within that event loop, set it to the running execution context. Then you call EnqueueJob("ScriptJobs", ScriptEvaluationJob, « sourceText »).
- From the running execution context, the ES spec will create a PendingJob record whose [[Realm]] is set appropriately. From that [[Realm]], we infer (although it is never explicitly stated) that the appropriate "ScriptJobs" queue to enqueue ScriptEvaluationJob into is that of the ECMAScript implementation containing pendingJobRecord.[[Realm]].

With continents:

- HTML has event loops, which are associated one-to-one with ES continents.
- When you want to run some script, you find an appropriate execution context within that event loop, set it to the running execution context. Then you call EnqueueJob("ScriptJobs", ScriptEvaluationJob, « sourceText »), noting that the "ScriptJobs" queue will be the one belonging to the event loop's continent.

Maybe it is not a big deal, and people can figure out the first version. The second version makes it a lot clearer to me though.